BackEnd/src/main/kotlin/cub/trackmyoffer/FeatureProviderRouting.kt

```diff
@@ -31,6 +75,373 @@ fun Route.featureProviderRouting(httpClient: HttpClient, config: FeatureProvider
                 val response = httpClient.get("${config.remote}/")
                 call.respondText(response.bodyAsText(), status = response.status)
             }
+
+            post("/profile") {
+                val profileReq = call.receive<ProfileData>()
+
+                profileReq.id = extractUserId(call)
+
+                val remoteResponse: HttpResponse = httpClient.post("${config.remote}/api/profile") {
+                    contentType(ContentType.Application.Json)
+                    setBody(profileReq)
+                }
+
+                val text = remoteResponse.bodyAsText()
+                call.respond(
+                    status = remoteResponse.status,
+                    message = text
+                )
+            }
+            post("/DEBUG/profile") {
+                val profileReq = call.receive<ProfileData>()
+
+                profileReq.id = 1//extractUserId(call)
+
+                val remoteResponse: HttpResponse = httpClient.post("${config.remote}/api/profile") {
+                    contentType(ContentType.Application.Json)
+                    setBody(profileReq)
+                }
+
+                val text = remoteResponse.bodyAsText()
+                call.respond(
+                    status = remoteResponse.status,
+                    message = text
+                )
+            }
+
+            get("/profile") {
+                val userId = extractUserId(call)
+
+                val remoteResponse: HttpResponse = httpClient.get("${config.remote}/api/profile/${userId}")
+
+                call.respond(
+                    status = remoteResponse.status,
+                    message = remoteResponse.bodyAsText()
+                )
+            }
+            get("/DEBUG/profile") {
```

Comment:
I'd suggest moving all debug endpoints under a separate route and allow disabling it based on config property, so we can disable it on prod

Reply:
Yep, a great idea, I just wanted to do something quick and "copy-paste" was quick enough of a deal